dev-tools/Procfile.dev

```diff
@@ -1,10 +0,0 @@
-server: npm run dev-start
```

Comment:
Yes, rethink is running as a separate daemon process in this scenario, never stops.